website/docs/components/form/radio-card/partials/code/component-api.md

```diff
@@ -47,7 +47,7 @@
 
 <Doc::ComponentApi as |C|>
   <C.Property @name="<[R].Icon>">
-    Yields an icon inside the card container. For details about its API check the [`FlightIcon`](https://flight-hashicorp.vercel.app/engineering) component.
+    Yields an icon inside the card container. For details about its API check the [`FlightIcon`](/foundations/icons/) component.
```

Comment:
```suggestion
    Yields an icon inside the card container. For details about its API check the [`FlightIcon`](/foundations/icons?tab=code) component.
```

I think this will work and take them directly to the code tab?